unlock-app/src/__tests__/components/content/AccountContent.test.tsx

```diff
@@ -144,4 +145,40 @@ describe('AccountContent', () => {
       })
     })
   })
+
+  describe('getStripeHelper', () => {
+    it('should do nothing when window does not contain stripe', () => {
+      expect.assertions(2)
+
+      jest.useFakeTimers()
+      const interval = setInterval(() => {}, 15000)
+      const setStripe = jest.fn()
+
+      getStripeHelper({}, interval, setStripe)
+
+      expect(clearInterval).not.toHaveBeenCalled()
+      expect(setStripe).not.toHaveBeenCalled()
+    })
+
+    it('should setStripe and clearInterval when the window does contain stripe', () => {
+      expect.assertions(2)
+
+      jest.useFakeTimers()
+      const interval = setInterval(() => {}, 15000)
+      const setStripe = jest.fn()
+      const Stripe = jest.fn()
+      ;(Stripe as any).version = 16000
```

Comment:
Meh. this syntax is weird :/

Reply:
🤔 we could start using semicolons at the end of every line